src/components/ImageView/index.js

```diff
@@ -179,6 +195,8 @@ class ImageView extends PureComponent {
                         containerHeight: height,
                         containerWidth: width,
                         zoomScale: scale,
+                    }, () => {
+                        this.setImageRegion(imageWidth, imageHeight);
```

Comment:
Also, why we need this?

Reply:
when the container is loaded or changed we need to update the image dimension to the container dimension.

Reply:
I just assumed the image is loaded before the container and that's made a white screen or image not get resized to the correct size. since we do not have the container dimension. 

but I think now this will not happen since we call this function on the component did mount and the container should be mounted.

Reply:
@rushatgabhane  make many tests without it and not have any issue so i delete it as you recommend